library/js/prescription-autocomplete/prescription-autocomplete.js

```diff
@@ -0,0 +1,138 @@
+ *
+ *
+ */
+
+
+
+$(document).ready(function () {
+    $("#sendScript").toggle();
+    $('#pharmacy-autocomplete').autocomplete({
+
+        source: function (request, response) {
+            // Fetch data
+            let autocompleteUrl = $("#pharmacyselect").data('autocomplete-url');
+            $.ajax({
+                url: autocompleteUrl+'?term='+request.term,
+                type: 'GET',
+                dataType: "json",
+                data: {
+                    search: request.term
+                },
+                success: function (data) {
+                    var resp = $.map(data,function(obj){
+                        //console.log(obj.id);
+                        var   phadd = obj.id + ' ' +obj.name + ' ' + obj.line1 + ' ' +obj.city;
+                        return phadd;
+                    });
+                    response(resp);
+                }
+            });
+        }
+    });
+
+});
+
+let p = document.getElementById('savePharmacy');
+p.addEventListener('click', savePharm);
+
+function savePharm(e) {
+    top.restoreSession();
+    let pharmId = document.getElementById('pharmacy-autocomplete').value;
+    let id = pharmId.split(" ");
+    let setPharm = "../../src/Rx/Weno/PatientPharmacyController.php";//$("#savePharmUrl").data('savePharm-url');
+    if (id[0] > 0) {
+        $.ajax({
+            url: setPharm+'?id='+id[0],
+            type: 'GET',
+            dataType: 'json',
+            success: function (result) {
+                $("#savePharmacy").toggle();
+                getReturnJson(result)
+            },
+            error: function(xhr, status, error){
+                console.log(xhr);
+                console.log(status);
+                console.log(error);
+                console.warn(xhr.responseText);
+            }
+        });
+        e.preventDefault();
+        $("#sendScript").toggle();
+        return false;
+    } else {
+        $("#savePharmacy").toggle();
+        e.preventDefault();
+        $("#sendScript").toggle();
+    }
+}
+
+let s = document.getElementById('sendScript');
+s.addEventListener('click', sendScripts);
+function sendScripts(e) {
+    top.restoreSession();
+    //While the user is waiting for the return response show this
+    $('#success').html("<i class='fa fa-refresh fa-spin fa-3x fa-fw'></i>");
+    //use ajax to get the list of prescriptions in the database
+    let scripts = $('#prescriptIds').data('ids');
+    //decided this time to send the prescriptions from the server instead of the browser
+    //that is why only the prescription id is needed to be transmitted
+    let sendRx = "../../interface/weno/transmitRx.php";
+    let request = [];
+    let responses = [];
+    //Make sure there has been at least one id passed
+    if (scripts) {
+        request.push(
+            $.ajax({
+                url: sendRx+'?scripts='+scripts,  //send a list of id's to the server to build the rx message and send
```

Comment:
See my comment at top of this PR. Is this script meant to send 1 or more than 1 script?

Reply:
more than one.

Reply:
it can send a comma-separated list